imagine/imagine.py

```diff
@@ -119,8 +119,8 @@ def _try_create_directory(directory: str) -> NoReturn:
     directory : string
         A ``string`` of a path pointing to a directory to attempt to create.
     """
-    if not os.path.exists(directory):
-        os.mkdir(directory)
+    
```

Comment:
This extra blank line is causing the CI tests to fail. Removing this line should clean that up.